salt/covid19/public.sls

```diff
@@ -0,0 +1,33 @@
+{% from 'lib.sls' import apache, create_user %}
+
+
+include:
+  - apache
+  - core
```

Comment:
Why include core? It's already included in top.sls, no?

Reply:
i am provisioning and deploying react app (i.e. covid19 explorer public) separately by command`./run.py ‘covid19-dev’ state.apply covid19.public ` without including core in public.sls it fails with git package missing error, that’s why i included core.init in public.sls

Reply:
You can deploy as state.apply core,covid19-public instead of including core.

Reply:
got it thanks @jpmckinney